specs-go/v1/descriptor.go

```diff
@@ -35,6 +35,9 @@ type Descriptor struct {
 	// Annotations contains arbitrary metadata relating to the targeted content.
 	Annotations map[string]string `json:"annotations,omitempty"`
 
+	// Data is an embedding of the targeted content.
```

Comment:
Might want a little bit more here about how its encoded (automatically through json package, etc.) in base64. A fetch can be avoided if the data field is present.

Reply:
Pushed another commit with more godoc, please take another look.